README.md

```diff
@@ -1,12 +1,12 @@
 # FeedHenry Android SDK
 
-[![Travis](https://img.shields.io/travis/feedhenry/fh-android-sdk.svg)](http://travis-ci.org/feedhenry/fh-android-sdk)
+[![Travis](https://img.shields.io/travis/feedhenry/fh-android-sdk/master.svg)](http://travis-ci.org/feedhenry/fh-android-sdk)
```

Comment:
shouldn't the image be gradle.svg instead of master.svg as the PR is against gradle branch? or gradle branch will be merged into master later?